axolotl-web/package.json

```diff
@@ -68,7 +68,7 @@
   "browserslist": [
     "> 1%",
     "last 2 versions",
-    "chrome 69"
+    "chrome 69"//support for chromium version used by QtWebEngine 5.12.
```

Comment:
The rule is no comments in json ;)
` npm ERR! JSON.parse Unexpected token "/" (0x2F) in JSON at position 2162 while parsing near "...ns\",\n    \"chrome 69\"//support for chromi..."`

Reply:
Ahh, for some reason I thought this is a js file.
There is no way to leave a message there for the future, is it?

Reply:
Found a way. :)